README.md

```diff
@@ -240,6 +244,49 @@ The PHP SDK is a robust and flexible library capable of integration with many ty
+
+Note that other "stateless" strategies such as `api` and `management` do not persist state data between requests, and therefor session storage configuration is not required.
+
+#### Cookies (Default)
+
+By default, the SDK will use cookies stored on an end-user's device to persist state data between requests. This data is stored in an encrypted state on the device. When using this storage method, the SDK will use the following configuration options to customize storage behavior:
+
+- `cookieSecret` (a string) to derive an encryption key for the session cookie.
+  - This should be a long, unique string that is ideally cryptographically generated. For example, running `openssl rand -hex 64` from a shell would deliver a viable string for this use.
+- `cookieDomain` (a string; falls back to your environment's `HTTP_HOST` if none is provided) to determine the domain to use for the session cookie.
+  - In most cases, this should be the domain of your application. To make cookies visible on all subdomains then the domain must be prefixed with a dot, like '.example.com'.
+- `cookieExpires` (an integer; defaults to `0`) to determine the expiration time (in seconds) for the session cookie.
+  - If set to `0` the cookie will expire at the end of the session (usually [but not always] when the browser closes).
+- `cookiePath` (a string; defaults to `/`) to determine the path to use for the session cookie.
+  - This is useful if you host multiple application instances on the same domain, at different paths.
+- `cookieSecure` (a boolean; defaults to `false`) to determine whether the session cookie should only be sent over secure connections.
+  - In production environments you should set this to `true`, unless you have a very good reason not to and [understand the risks](https://owasp.org/www-community/controls/SecureCookieAttribute).
+
+It's important to configure these values properly for your application to ensure session cookies are stored securely and accessible to your application, especially once it is pushed to production.
+
+Cookies are stored in an encrypted state using the `cookieSecret` as a salt, and it is therefor important to keep this value secure. Treat it like a password. Because of the size of these encrypted cookies can sometimes grow relatively large (varying based on the size of the user data), the SDK uses a technique called "chunking" to avoid hitting size limitations imposed by some browsers. The SDK will manage these chunked cookies (ending in a suffix of _1, _2, and so on) efficiently. Note that some legacy browsers that have reached end-of-life, particularly Internet Explorer, may have trouble with chunked cookies.
```

Comment:
```suggestion
Cookies are stored in an encrypted state using the `cookieSecret` as a salt, and it is therefore important to keep this value secure. Treat it like a password. Because of the size of these encrypted cookies can sometimes grow relatively large (varying based on the size of the user data), the SDK uses a technique called "chunking" to avoid hitting size limitations imposed by some browsers. The SDK will manage these chunked cookies (ending in a suffix of _1, _2, and so on) efficiently. Note that some legacy browsers that have reached end-of-life, particularly Internet Explorer, may have trouble with chunked cookies.
```